streams/nl_rakuten.m3u

```diff
@@ -0,0 +1,7 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies Netherlands (720p)
+https://rakuten-comedymovies-5-eu.rakuten.wurl.tv/playlist.m3u8
```

Comment:
It actually is German, sorry for confusing both languages. Felt weird that Germany had 2 slots.